common/lib/xmodule/xmodule/tests/test_content.py

```diff
@@ -103,19 +115,59 @@ def test_thumbnail_none(self):
     )
     @ddt.unpack
     def test_generate_thumbnail_image(self, original_filename, thumbnail_filename):
-        contentStore = ContentStore()
+        content_store = ContentStore()
         content = Content(AssetLocation(u'mitX', u'800', u'ignore_run', u'asset', original_filename), None)
-        (thumbnail_content, thumbnail_file_location) = contentStore.generate_thumbnail(content)
+        (thumbnail_content, thumbnail_file_location) = content_store.generate_thumbnail(content)
         self.assertIsNone(thumbnail_content)
-        self.assertEqual(AssetLocation(u'mitX', u'800', u'ignore_run', u'thumbnail', thumbnail_filename), thumbnail_file_location)
+        self.assertEqual(
+            AssetLocation(u'mitX', u'800', u'ignore_run', u'thumbnail', thumbnail_filename),
+            thumbnail_file_location
+        )
+
+    @patch('xmodule.contentstore.content.Image')
+    def test_image_is_closed_when_generating_thumbnail(self, image_mock):
+        # We used to keep the Image's file descriptor open when generating a thumbnail.
+        # It should be closed after being used.
+        mock_image = [None]
+
+        def image_open_mock(_):
+            """mock Image.open implementation"""
+            mock_image[0] = MockImage()
+            return mock_image[0]
+        image_mock.open = image_open_mock
```

Comment:
Hmm, I've noticed this can be simplified by simply using `image_mock.open.return_value`. @cahrens do you think it's worth still improving, even though it's already after your approval? (I have the diff ready, but didn't want to push any non-pure-quality changes without notifying anyone.)